src/esp/gfx/GenericDrawable.cpp

```diff
@@ -54,14 +56,18 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
     lightColors.emplace_back((*lightSetup_)[i].color);
   }
 
+  Mn::Color3 highLightColor = 0xa5c9ea_rgbf * 0.5;
   (*shader_)
-      .setAmbientColor(materialData_->ambientColor)
+      .setAmbientColor(selected_ ? highLightColor : materialData_->ambientColor)
       .setDiffuseColor(materialData_->diffuseColor)
       .setSpecularColor(materialData_->specularColor)
       .setShininess(materialData_->shininess)
       .setLightPositions(lightPositions)
       .setLightColors(lightColors)
-      .setObjectId(materialData_->perVertexObjectId ? 0 : node_.getId())
+      // e.g., semantic mesh has its own per vertex annotation, which has been
+      // uploaded to GPU so simply pass 0 to the uniform "objectId" in the
+      // fragment shader
+      .setObjectId(materialData_->perVertexObjectId ? 0 : drawableId_)
```

Comment:
Possible un-intended consequences of this change. This will cause all `SEMANTIC` sensor output for objects added through the physics interface (or without `perVertexObjectId`) to be the unique DrawableId. 


**Replica before** (note the scene disappears because replaced node_.getId() for chairs was default -1)
<img width="50%" height="50%" src="https://user-images.githubusercontent.com/1445143/86168119-8e32a580-bacc-11ea-9459-d4055f595d7f.jpg">

**Replica after** (now the chairs are rendered with DrawableId here)
<img width="50%" height="50%" src="https://user-images.githubusercontent.com/1445143/86168128-91c62c80-bacc-11ea-93c2-db83c6c78c0f.jpg">

This will also cause scenes without semantic annotations to render DrawableId through the `SEMANTIC` sensor.

**Van-gogh-room before** (was all -1 since no semantics are configured)
<img width="50%" height="50%" src="https://user-images.githubusercontent.com/1445143/86169203-1cf3f200-bace-11ea-8a70-6f8d7757fee7.jpg">

**Van-gogh-room after** (now all rendered as DrawableId)
<img width="50%" height="50%" src="https://user-images.githubusercontent.com/1445143/86169212-20877900-bace-11ea-8fb5-90568bf58b8a.jpg">